releasetool/commands/tag/python.py

```diff
@@ -56,44 +56,37 @@ def determine_release_tag(ctx: TagContext) -> None:
             f"head ref {head_ref}."
         )
         ctx.release_tag = click.prompt(
-            "What should the release tag be (for example, storage-1.2.3)?"
+            "What should the release tag be (for example, v1.2.3)?"
         )
 
     click.secho(f"Release tag is {ctx.release_tag}.")
 
 
 def determine_package_name_and_version(ctx: TagContext) -> None:
     click.secho("> Determining the package name and version.", fg="cyan")
-    if "google-cloud-python" in ctx.upstream_repo:
-        match = re.match(
-            r"(?P<name>.+?)-(?P<version>\d+?\.\d+?(\.\d+)+)", ctx.release_tag
-        )
-        ctx.package_name = match.group("name")
-        ctx.release_version = match.group("version")
-        click.secho(
-            f"Package name: {ctx.package_name}, "
-            f"package version: {ctx.release_version}."
-        )
-    else:
-        match = re.match(r"(.+)?(?P<version>\d+?\.\d+?(\.\d+)+)", ctx.release_tag)
-        ctx.release_version = match.group("version")
-        click.secho(f"Package version: {ctx.release_version}.")
+
+    match = re.match(r"(.+)?(?P<version>\d+?\.\d+?(\.\d+)+)", ctx.release_tag)
```

Comment:
note: in `\d+?\.`, the question mark is useless because minimal matching quantifier will behave in the same way as the regular quantifier in this case (when followed by a non-`\d` such as a dot).

Reply:
Also please note that the first `.+` will eat as many characters as possible. E.g. if a string is `v1.2.3.4`, the first `(.+)` will consume `v1.`, `version` will be `2`, and the trailing part will be `.3.4`.

Reply:
Ah this was copied from the node regex, bu since you folks use a different method for finishing releases this is probably out of date/incorrect.

I'll take this back to the drawing board. Thank you for the review!

Reply:
Ignore my comment above, this is the wrong part of the file. 🙃